test/src/e2e_vm_tests/test_programs/should_pass/language/bitwise_not/src/main.sw

```diff
@@ -0,0 +1,14 @@
+script;
+
+use core::ops::*;
+use std::assert::assert;
+use std::logging::log;
```

Comment:
I think none of these are needed. `assert` is also in the prelude.

Reply:
Removed